charts/selenium-grid/values.yaml

```diff
@@ -398,6 +399,15 @@ chromeNode:
   #   persistentVolumeClaim:
   #     claimName: my-pv-claim
 
+  # Keda scaled object configuration
+  autoscalingEnabled: false
+  maxReplicaCount: 8
+  hpa:
+    url: http://selenium-hub.default:4444/graphql # Repalce your http graphql url here
```

Comment:
`selenium-hub.default` can this be retrieved from the Hub or Router service or pod name?

Reply:
I had the same thought too, but created the PR in the excitement of sharing it😁. Will add this too.